setup

```diff
@@ -13,12 +20,13 @@ if not os.path.exists(requirements):
 
 pip.main(['install', '--upgrade', '-r', requirements])
 
-if which('unzip') == None:
+if which('unzip') is None:
     print('The following executables are needed and were not found: unzip')
 
 print('Downloading datasets (this might take several minutes depending on your internet connection)')
 
 # This is imported here because serenata_toolbox is only installed when pip.main is called (line 9)
 from serenata_toolbox.datasets import fetch_latest_backup
 os.makedirs('research/data', exist_ok=True)
-fetch_latest_backup('research/data')
+fetch_latest_backup('research/data', force_all=args.all)
```

Comment:
We have some contributors that code on windows, would be nice if the path to the `data` directory was built using `os.path.join` to avoid breaking the code:

fetch_latest_backup(os.path.join('research', 'data'), force_all=args.all)